statchatui/src/Stats/Stats.jsx

```diff
@@ -1,25 +1,29 @@
 import React from "react";
 
-const Stats = ({ statData }) => {
+const Stats = ({ statData, data }) => {
   if (!statData) {
-    return <div>Stats not yet fetched.</div>;
+    return <div>{data || "Stats not yet fetched."}</div>;
```

Comment:
Why do we print out data here? What is `data`?

Reply:
i just deleted it, i dont need that.

Reply:
i was going to have data be an error but i forgot what i was doing so ill delete it